python/datashaper/tests/test_schema.py

```diff
@@ -25,42 +25,28 @@
 
 
 @cache
-def load_inputs(pandas_dtype_backend: PandasDtypeBackend, engine: str):

```

Comment:
So the issue here is that the tests aren't compatible with the Pyarrow backend anymore right?

Reply:
Correct - pyarrow requires jsonl, but regular engine (ujson) doesn't support it. I had to choose one, and default pandas made sense until pyarrow is 100% supported as an engine, and we may end up just using that for polars anyway.